sysinit/coqargs.ml

```diff
@@ -231,7 +231,7 @@ let set_option = let open Goptions in function
   | opt, OptionAppend v -> set_string_option_append_value_gen ~locality:OptLocal opt v
 
 let get_compat_file = function
-  | "8.21" -> "Stdlib.Compat.Coq821"
+  | "9.0" -> "Stdlib.Compat.Coq90"
```

Comment:
For now, it is easy to direct to `Rocq90` here.
Obviously, this clashes with your work in #19370, but you are already aware of this.